celery/contrib/testing/worker.py

```diff
@@ -119,7 +119,7 @@ def _start_worker_thread(app,
         logfile=logfile,
         # not allowed to override TestWorkController.on_consumer_ready
         ready_callback=None,
-        without_heartbeat=True,
+        without_heartbeat=kwargs.pop("without_heartbeat", True),
```

Comment:
this isn't going to break the existing API write? shouldn't this overridable feature mentioned in the docs? and can we provide some sort of unit tests?

Reply:
The default value is False, so this should be backwards incompatible.

Reply:
can you add documentation for this?

Reply:
Yes, probably next week.